google/cloud/spanner/commit_options.h

```diff
@@ -94,12 +95,23 @@ class CommitOptions {
     return transaction_tag_;
   }
 
+  CommitOptions& set_max_commit_delay(
```

Comment:
The other getters/setters, for better or worse, all have a single-line comment.  We might as well continue that for now.